src/frontend-admin/src/components/App/App.module.css

```diff
@@ -16,3 +16,105 @@
 .appLink {
   color: #61dafb;
 }
+
+.navContainer {
+  background: white;
+  height: 140px;
+}
+
+nav {
```

Comment:
In general I recommend not adding styles directly to element types at the top level like this - instead I would put a classname on the nav tag and reference that class name here instead. The reason has to do with how css scopes.